detox/src/devices/drivers/android/emulator/EmulatorDriver.js

```diff
@@ -82,6 +86,7 @@ class EmulatorDriver extends AndroidDriver {
   }
 
   async shutdown(deviceId) {
+    await this.instrumentation.setTerminationFn(null);
```

Comment:
It is useless to fire the termination callback when the device itself is shutting down — only raises chances for badly executed/aborted `adb shell` commands in the middle.